### PR TITLE
Fix typo on committee page

### PR DESCRIPTION
**Pull request by NyKo24**

### CODE REVIEW

File: templates/coordinator/_committee_list.html.twig [hunk 1]
```diff
@@ -28,7 +28,7 @@
             <td>
                 {% for messages in app.flashes(["error_#{committee.id}"]) %}
                     {% for message in messages %}
-                        <div class="form__errors">
```
Comment (marineb) [inline templates/coordinator/_committee_list.html.twig]: Why are you changing this?

Comment (NyKo24) [reply templates/coordinator/_committee_list.html.twig]: because the class name definition in CSS is form__error

Comment (marineb) [reply templates/coordinator/_committee_list.html.twig]: I don't believe so. There is a `form__error` and a `form__errors`. Look for `__errors` and you'll find it.

File: templates/coordinator/_layout.html.twig [hunk 2]
```diff
@@ -11,9 +11,9 @@
                     {{ block('page_title') }}
                 </h1>
                 <p class="text--center">
-                    L’espace coordinateur vous permet d'apprecier les comités des
+                    L’espace coordinateur vous permet d'apprécier les comités des
                     zones que vous gérez. Seule l’équipe du QG sera en mesure de
-                    voir vos appréciations.
+                    voir vos&nbsp;appréciations.
```
Comment (marineb) [inline templates/coordinator/_layout.html.twig]: `&nbsp;` ?

**Pull request merged**